model/metrics/messaging-metrics.yaml

```diff
@@ -0,0 +1,80 @@
+groups:
+  - id: metric.messaging.attributes
+    type: attribute_group
+    stability: experimental
+    brief: "Common messaging metrics attributes."
+    extends: attributes.messaging.common.minimal
+    attributes:
+      - ref: messaging.system
+        requirement_level: required
+      - ref: messaging.destination.partition.id
+      - ref: messaging.destination.name
+        requirement_level:
+          conditionally_required: if and only if `messaging.destination.name` is known to have low cardinality. Otherwise, `messaging.destination.template` MAY be populated.
+      - ref: messaging.destination.template
+        requirement_level:
+          conditionally_required: if available.
+
+  # durations
+  - id: metric.messaging.client.request.duration
+    type: metric
+    metric_name: messaging.client.request.duration
+    brief: "Duration of messaging operation initiated by a producer or consumer."
+    note: >
+      This metric SHOULD NOT be used to report processing duration - processing duration is reported in `messaging.process.duration` metric.
```

Comment:
```suggestion
      This metric SHOULD NOT be used to report processing duration - processing duration is reported in `messaging.client.process.duration` metric.
```

Reply:
we kept the same name for `messaging.process.duration` (due to concern over `client.process`). We can revisit the naming for it.

Reply:
oh, I don't know what I was thinking, I agree with `messaging.process.duration` 👍

Reply:
since `messaging.process.duration` is not a "client" metric, wdyt of renaming `messaging.client.request.duration` to `messaging.client.operation.duration` (it is all "client" operations, which doesn't include process operations)

Reply:
I like it. @pyohannes any objections?